contrib/opencensus-ext-azure/examples/metrics/simple.py

```diff
@@ -38,7 +38,7 @@
 def main():
     # Enable metrics
     # Set the interval in seconds in which you want to send metrics
-    exporter = metrics_exporter.new_metrics_exporter(export_interval=2)
+    exporter = metrics_exporter.new_metrics_exporter(export_interval=2.0)
```

Comment:
I suggest that we give a reasonable exporter interval comparable to the ingestion and processing latency.